Mapper/BaseGroupedMapper.php

```diff
@@ -257,6 +263,11 @@ public function hasOpenTab()
     abstract protected function setTabs(array $tabs);
 
     /**
+     * @return string
+     */
+    abstract protected function getContext();
```

Comment:
@greg0ire The "getContext" method already exists in the `FormMapper` and `ShowMapper` classes. Do you want me to rename it everywhere ?

@Soullivaneuh Should I comment this method in the `BaseGroupedMapper` class only ?